### DISCUSSION TITLE
Has anyone added support for Sinope WL4210?

### DISCUSSION THREAD

**Discussion by bmf7777** [Koenkk/zigbee2mqtt#18967] [Devices]
it's similar to WL4200S with an additional water contact on the base unit as well as external probe. i tried to add the device locally without much luck.

**Comment by bmf7777** [comment]
i duped the code for WL4200S `in` sinope.js ... for the time being they are the same device with an addition detector in the base unit

**Comment by ghost** [comment]
I've just got to pair my WL4210 and did the same as you, copy/pasted the code for the WL4200S. Seems to work fine for me too

**Comment by audricschiltknecht** [comment]
PR opened https://github.com/Koenkk/zigbee-herdsman-converters/pull/6842
